Document/0x04f-Testing-Network-Communication.md

```diff
@@ -560,25 +560,21 @@ Identify all of the tested application's critical operations (e.g., user enrollm
 
```

Comment:
I'm not sure that the reference to Android 10 changes at line 549 is still relevant

Reply:
That's a good catch. That reference is there because it's mentioned in the text above. Going forward we'll remove all "References" sections and generate them automatically. But for now let's leave all current references there since we know that some only exist in the "References" section so we have to ensure that they are in the text as well.